### ISSUE TITLE
`single-line-block-comment-style`: Support single-line JSDoc

### ISSUE THREAD

**Issue by krislintigo** [sindresorhus/eslint-plugin-unicorn#3723]
### Description

JSDoc supports single-line descriptions, but currently they are formatted to multi-line

### Examples

```js
// ✅ Should be valid
/** Single-line JSDoc */

// ❌ still incorrect, because this is not JSDoc
/* Single-line block comment */
```

**Comment by sindresorhus** [comment]
This is already supported with the `single-line` option:

```js
'unicorn/single-line-block-comment-style': ['error', 'single-line'],
```

Note that the option applies to all block comments, so `/* Single-line block comment */` is valid too. We are not going to treat JSDoc as a special case.
